src/Dapr.Actors/Runtime/ActorTimer.cs

```diff
@@ -48,12 +48,12 @@ internal async Task<string> SerializeAsync()
 
             writer.WriteStartObject();
 
-            if (this.DueTime != null)
+            if (this.DueTime != null && this.DueTime >= TimeSpan.Zero)
```

Comment:
dueTime will always be >=0, Reminders do a check,(https://github.com/dapr/dotnet-sdk/blob/master/src/Dapr.Actors/Runtime/ReminderInfo.cs#L93). There should be a similar check for Timer dueTime also.